CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.5.0 (Sep 25, 2019)
+FEATURES:
+* [30](https://github.com/nginxinc/nginx-plus-go-client/pull/30): *Additional Upstream Server Parameters*.
+The client now supports configuring `route` `backup` `down` `drain` `weight` `service` in upstream.
+* [31](https://github.com/nginxinc/nginx-plus-go-client/pull/31): *Support `location zones` and `resolver` metrics*.
+
+BUGFIXES:
+* [29](https://github.com/nginxinc/nginx-plus-go-client/pull/29): *Fix `MaxFails` in UpstreamServer and StreamUpstreamServer and improved test coverage*. Previously the default was `0` as opposed to the intended `1`.
```

Comment:
Is test coverage improvement is relevant to changelog? 

```suggestion
* [29](https://github.com/nginxinc/nginx-plus-go-client/pull/29): *Fix `MaxFails` in UpstreamServer and StreamUpstreamServer*. Previously the default was `0` as opposed to the intended `1`.
```

Reply:
```suggestion
[29](https://github.com/nginxinc/nginx-plus-go-client/pull/29): *Fix max_fails parameter in upstream servers*. Previously, if the MaxFails field was not explicitly set, the client would incorrectly configure an upstream with the value `0` instead of the correct value `1`.
```